helm/resource_release_test.go

```diff
@@ -714,6 +758,82 @@ func TestGetValues(t *testing.T) {
 	}
 }
 
+func TestGetValuesString(t *testing.T) {
+	d := resourceRelease().Data(nil)
+	d.Set("set", []interface{}{
+		map[string]interface{}{"name": "foo", "value": "42", "type": "string"},
+	})
+
+	values, err := getValues(d)
+	if err != nil {
+		t.Fatalf("error getValues: %s", err)
+		return
+	}
+
+	if err != nil {
+		t.Fatalf("error parsing returned yaml: %s", err)
+		return
+	}
+
+	if values["foo"] != "42" {
+		t.Fatalf("error merging values, expected %q, got %s", "42", values["foo"])
+	}
+}
+
+func TestCloakSetValues(t *testing.T) {
```

Comment:
Given that these functions are private to the package and are an implementation detail of the helm_release resource, I think it would have been fine to test this function by creating a few different incantations of the resource in the acctest above. 

I can see why you wanted to test this in isolation though, so I'm happy to leave it in.